index.html

```diff
@@ -1,56 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
-    <link rel="stylesheet" href="./style.css" type="text/css">
-    <title>Portfolio Homepage</title>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet" />
+  <link rel="stylesheet" href="./style.css" type="text/css" />
+  <title>Portfolio Homepage</title>
 </head>
+
 <body>
-    
   <!-- Navigation Bar Section -->
 
   <div class="wrapper">
     <div class="front-wrapper">
       <header>
-        <nav class="flex-nav">
-          <a href="https://yussufob.github.io/My-Portfolio-Part-II/" class="my-logo">Y-Bash</a>
-          <a href="https://yussufob.github.io/My-Portfolio-Part-II/" class="union-icon"><img src="./images/Union.svg" alt="list-icons"></a>
-          <ul class="menu-list">
-            <li><a href="https://yussufob.github.io/My-Portfolio-Part-II/">Portfolio</a></li>
-            <li><a href="https://yussufob.github.io/My-Portfolio-Part-II/">About</a></li>
-            <li>Contact</li>
+        <nav>
+          <div class="logo">Bash</div>
+          <div class="openMenu"><img src="./images/Union.svg" alt="hamburger"></div>
+          <ul class="mainMenu">
+            <li class="shieldOff1"><a href="#my-portfolio">Portfolio</a></li>
+            <li class="shieldOff2"><a href="#about-myself">About</a></li>
+            <li class="shieldOff3"><a href="#contact-me">Contact</a></li>
```

Comment:
- [ ] Please make sure that the menu options are aligned left as found in the Figma design. 

Your Implementation | Figma Design
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/161729541-614a23ea-cd96-45be-ae72-c6afbba5a6af.png) | ![](https://user-images.githubusercontent.com/63087491/161729610-5e0aa4a6-a3b2-49a3-b92f-7678e144e7cc.png)

Reply:
Hi @YussufOB , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._